prow/README.md

```diff
@@ -163,7 +163,7 @@ Jenkins' URL, such as `http://pull-jenkins-master:8080`.
 6. Create a namespace for test pods named "test-pods".
 
  ```
- kubectl create -f cluster/test_pods_namespace.yaml
+ kubectl create -f cluster/test_pod_namespace.yaml
```

Comment:
Great. I was wondering about the namespace file.
I've noticed that the namespace creation is missing from the Makefile as well.